src/Command.js

```diff
@@ -31,18 +31,17 @@ module.exports = BotModule.extend({
             if (command === this.commandName || this.commandAliases.includes(command)) {
                 let timeOut = this.config.messageRemoveDelay;
                 let promise = this.processMessage(message, tokens);
-                
+
                 if (!this.shouldDeleteMessage) {
                     return;
                 }
-                
+
                 message.delete(timeOut).catch(err => {/*do nothing*/});
-                
+
                 promise.then(messages => {
                     this.deleteMessage(messages, timeOut);
                 }).catch(error => {
                     console.error(error.message);
-                    done();
```

Comment:
The function "done" doesn't exist anywhere in this context. It threw errors about it not existing when I caused this promise to throw an error.